frontend/src/pages/detail/PODetail.tsx

```diff
@@ -63,18 +53,19 @@ const columns: TableColumn[] = [
   },
 ];
 
-class ModifyPOPage extends React.Component<{}, modifyPOState> {
-  constructor(props = {}) {
-    super(props);
-    this.state = {
-      date: new Date(),
-      vendor: "asdfa",
-      isModifiable: false,
-      isConfirmationPopVisible: false,
-    };
-  }
-
-  onCellEditComplete = (e: {
+export default function PODetail() {
+  const location = useLocation();
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const detailState = location.state! as PODetailState;
```

Comment:
Again same as previous comment. I have done a workaround on my branch which I will remove to remain consistent, but this issue can be resolved throuhg page feedthrough implementation.

Reply:
As above, already implemented